page.css

```diff
@@ -0,0 +1,98 @@
+
+html,
+body {
+    background-color: #FFE1E9;
+    font-family: sans-serif;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+@media only screen and (min-width: 375px) {
+
+    .body__image {
+        display: flex;
+        height: 150px;
+
+    }
+    
+    .page_container {
+        display: block;
+        background-color: white;
+    }
+    
+    
+    .body_maincard {
+        background-color: white;
+    }
+    
+    .body__top_layout {
+         background-color: white;
+         padding: 50px;
+    
+    }
+    
+    p {
+        font-size: 33px; 
+    }
+
+
+    h1 {
+    display: block;
+    }
+
+    textarea {
+        border-radius: 15px;
+        width: 99%;
+        font-size: 35px;
+        text-align: center;
+        line-height: 80px;
+        height:100px;
+    }
+
+    .body__signedin {
+        padding: 30px;
+        font-size: 35px;
+        display: flex;
+    }
+    
+    input[type=checkbox] {
+        transform: scale(3.8);
+        margin-right: 30px;
+    
+    }
+
+    .body__new-number {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+    }
+    .sign__in {
+        border: 0;
+        line-height: 2.5;
+        padding: 0px 70px 0px 70px;
+        font-size: 3rem;
```

Comment:
why rem?

Reply:
I think it's better to use 'em' here. The parent of 'em' element will be responsible for its size so that will help us to scale font sizes on different resolutions in the future

Reply:
'rem' is not a good solution here